DateCalculator/js/pages/switcher.js

```diff
@@ -0,0 +1,24 @@
+import function timer from './timer'
```

Comment:
здесь ошибка в записи импорта, если хочешь импортировать функцию timer то надо вот так)
```suggestion
import { timer } from './timer'
```